packages/devops/scripts/deployment-common/configureNginx.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -le
+
+HOME_DIR=/home/ubuntu
+CONFIG_DIR=$HOME_DIR/tupaia/packages/devops/configs
+TEMPLATE_DIR=$CONFIG_DIR/nginx-template
+
+# configure nginx
+if [[ $DEPLOYMENT_NAME == "production" ]]; then
+  DOMAIN_PREFIX=""
+  SUBDOMAIN_PREFIX=""
+else
+  DOMAIN_PREFIX="$DEPLOYMENT_NAME."
+  SUBDOMAIN_PREFIX="$DEPLOYMENT_NAME-"
+fi
```

Comment:
This is a common block, but I would agree that there's quite a lot of factoring out going on here already and it might be more confusing if it was included from a common location

Reply:
Yeah it feels like it'd be a bit awkward to break this out into a script. Tho I feel I don't have a good intuition around how to keep things DRY with bash scripting... Might leave as is for now?